MobileWallet/TariLib/Wrappers/Utils/TestnetKeyServer.swift

```diff
@@ -93,7 +92,7 @@ class TestnetKeyServer {
 
         self.wallet = wallet
         self.signature = try wallet.signMessage(message)
-        self.url = URL(string: "\(SERVER)/free_tari/allocate_max/\(publicKeyHex)")!
+        self.url = URL(string: "\(TariSettings.shared.faucetServer)/free_tari/allocate_max/\(publicKeyHex)")!
```

Comment:
Cool change 👍  Something like `TariSettings.faucetServer` or `TariSettings.Network.faucetServer` would read even better. `shared` implies there could be other instances of the class.

Reply:
I got this from reading up on some design patterns in swift, `shared` always implies it's a singleton, so it's a shared instance and there can only be one. Like with `UIApplication.shared.delegate` that means this is the only instance of the `AppDelegate` class. Do you think maybe a singleton isn't a good pattern for settings in this case?

Reply:
I really don't know about the specific common pattern for this in Swift, but just by looking at it, an object instance might be redundant if there are no functions to be accessed but only constants.

Reply:
but there is one function for it, but to be honest when I first created that class I thought I would need more. Let me look at some other projects and see what pattern they used and we can chat further after.

Reply:
No probs:)